jmx-scraper/src/main/resources/jetty.yaml

```diff
@@ -0,0 +1,71 @@
+---
+
+rules:
+
+  - bean: org.eclipse.jetty.io:context=*,type=managedselector,id=*
+    mapping:
+      selectCount:
+        metric: jetty.select.count
+        type: counter
+        # Unit should not use a plural form, left as-is for compatibility with jetty.groovy
+        # https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units
+        unit: "{operations}"
```

Comment:
Is there a reason that we're trying to keep this compatibility instead of fixing it?

Reply:
Yes, I think this is better as a first step to stay very close to the original implementation (even if it's buggy).
The unit tests are almost copied as-is which helps us prevent any regressions and to ensure that we have the same behavior between the two implementations.

However, I think that it would be a good idea to remove this (and the other ones) as soon as possible with a separate PR that fixes the two implementations (this one in yaml and groovy) at once. This should not be a breaking change as the units are very unlikely to be relied on beyond documentation, the captured values remain the same.

Reply:
Giving it a second thought on this, the "we'll never be able to fix this if we perpetuate it" changed my mind and I've fixed that directly in this PR with 187ca89.

I was thinking more about the process of replicating existing tests with some fidelity than fixing small inconsistencies while we are at it when we can do it in a non-breaking way.